resources/base/crds/minio.min.io_tenants.yaml

```diff
@@ -694,6 +694,18 @@ spec:
                     type: integer
                   resources:
                     properties:
+                      claims:
```

Comment:
what caused this to be added?

Reply:
`ontroller-gen@v0.11.1` currently merged in master is adding the `claims` fields.
Claims are part of kubernetes 1.26 alpha  https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/